src/envoy/http/path_rewrite/filter.cc

```diff
@@ -112,13 +112,8 @@ FilterHeadersStatus Filter::decodeHeaders(RequestHeaderMap& headers, bool) {
         absl::StrCat("Request `", utils::readHeaderEntry(headers.Method()), " ",
                      utils::readHeaderEntry(headers.Path()),
                      "` is getting wrong route config"),
-        utils::generateRcDetails(
-            utils::kRcDetailFilterPathRewrite,
-            utils::kRcDetailErrorTypeWrongRouteConfig,
-            absl::StrCat(
-                "Request path: ", utils::readHeaderEntry(headers.Path()),
-                " is mismatched with url_template: ",
-                per_route->config_parser().url_template())));
```

Comment:
We may want to keep url_template  in rc_detail. There could be multiple routes matched with a path,  we don't know which route is selected.   By printing the url_template we will know

Reply:
ACK

Reply:
Done.